README.md

```diff
@@ -47,9 +47,9 @@ Create and activate a virtualenv with the development requirements:
 ## Testing
 
 The Python operator framework includes a very nice harness for testing
-operator behaviour without full deployment. Just `run_tests`:
+operator behaviour without full deployment. Just `unit` tox target:
```

Comment:
hmm, should the wording be "Just target the `unit` env via tox"? Or maybe "Just run:" since you are already sharing the CLI command below.

Reply:
You're right, this wording is not good.